tests/PersistentArrayTest.php

```diff
@@ -159,7 +158,7 @@ public function testInsert()
             ],
         ];
 
-        $p = new Persistence\Array_($a);
+        $p = new Persistence\Array_(new Persistence\Array_\ArrayByRef($a));
```

Comment:
If we move ArrayByRef to atk4/core Then things like this would be simpler:
`$p = new Persistence\Array_(new \atk4\core\ArrayByRef($a));` or maybe even use static method there something like:
`$p = new Persistence\Array_(\atk4\core\Helper::ArrayByRef($a));`